src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private final Color[] colors = Color.values();
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+
+        int index = random.nextInt(colors.length);
+        return String.valueOf(colors[index]);
```

Comment:
[Use name() for getting String representation of enum constants](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants)